cmd/main.go

```diff
@@ -255,6 +257,42 @@ func main() {
 		defer appProfileReconcilerController.StopController()
 	}
 
+	// Start the ClamAV scanner
```

Comment:
Isn't this block should be under a condition that the user enabled ClamAV?

Reply:
See the code below, I am just initiating the config of the clamav scanner but I check if the variables are actually set.